src/hammer-vlsi/hammer_vlsi/constraints.py

```diff
@@ -14,10 +14,11 @@
 from hammer_utils import reverse_dict
 from .units import TimeValue, VoltageValue, TemperatureValue
 
-__all__ = ['ILMStruct', 'SRAMParameters', 'Supply', 'BumpAssignment',
-           'BumpsDefinition', 'ClockPort', 'OutputLoadConstraint',
-           'DelayConstraint', 'ObstructionType', 'PlacementConstraintType',
-           'Margins', 'PlacementConstraint', 'MMMCCornerType', 'MMMCCorner']
+__all__ = ['ILMStruct', 'SRAMParameters', 'Supply', 'PinAssignment',
+           'BumpAssignment', 'BumpsDefinition', 'ClockPort',
+           'OutputLoadConstraint', 'DelayConstraint', 'ObstructionType',
+           'PlacementConstraintType', 'Margins', 'PlacementConstraint',
+           'MMMCCornerType', 'MMMCCorner']
```

Comment:
@edwardcwang Can we delete `__all__`? Is there any reason to hide some things? It's annoying to maintain.

Reply:
In this file pretty much everything in this file is exposed there anyway, so it's fine to do it in this file.